public/locales/en/translation.json

```diff
@@ -291,6 +292,13 @@
           "title": "Discard current modifications ?",
           "body": "You will lose all your current changes made on the scenario parameters."
         }
+      },
+      "about": {
+        "title": "Generic Webapp",
```

Comment:
Can we reuse the application title `Azure Sample Web Application` to stay consistent ?